peps/pep-0752.rst

```diff
@@ -259,180 +199,83 @@ If the following criteria are all true for a given upload:
 
-To allow installers and other tooling insight into this project-level metadata
-of a namespaced project, the :pep:`JSON API <691>` version will be incremented
-and support new keys for the project endpoint.
+The :pep:`JSON API <691>` version will be incremented from ``1.0`` to ``1.1``.
+The following API changes MUST be implemented by repositories that support
+this PEP. Repositories that do not support this PEP MUST NOT implement these
+changes so that consumers of the API are able to determine whether the
+repository supports this PEP.
+
+.. _project-detail:
 
-The ``owner`` key SHOULD be added and refer to the owner of the project,
-whether an organization or a user.
+Project Detail
+''''''''''''''
 
-The ``namespace`` key MAY be added and MUST be ``null`` if the project does not
-match an active namespace grant. If the project does match a namespace grant,
-the value MUST be a mapping with the following keys:
+The :pep:`project detail <691#project-detail>` response will be modified as
+follows.
 
-* ``name``: This is the associated `normalized <naming_>`_ namespace e.g.
+The ``namespace`` key MUST be ``null`` if the project does not match an active
+namespace grant. If the project does match a namespace grant, the value MUST be
+a mapping with the following keys:
+
+* ``prefix``: This is the associated `normalized <naming_>`_ namespace e.g.
   ``foo-bar``. If the owner of the project owns multiple matching grants then
   this MUST be the namespace with the most number of characters. For example,
   if the project name matched both ``foo-bar`` and ``foo-bar-baz`` then this
   key would be the latter.
-* ``owners``: This is an array of organizations that
-  `own <grant-ownership_>`_ the grant. This is useful for tools that wish to
-  make a distinction between official and community packages by checking if
-  the array contains the project ``owner``.
-* ``shared``: This is a boolean indicating whether the namespace is
-  `shared <shared-namespaces_>`_.
-
-The presence of the ``namespace`` key indicates support for this PEP.
+* ``authorized``: This is a boolean and will be true if the project owner
+  is an organization and is one of the current owners of the grant. This is
+  useful for tools that wish to make a distinction between official and
+  community packages.
+* ``open``: This is a boolean indicating whether the namespace is
+  `open <open-namespaces_>`_.
+
+Namespace Detail
+''''''''''''''''
+
+The format of this URL is ``/namespace/<namespace>`` where ``<namespace>`` is
+the `normalized <naming_>`_ namespace. For example, the URL for the namespace
+``foo.bar`` would be ``/namespace/foo-bar``.
+
+The response will be a mapping with the following keys:
+
+* ``prefix``: This is the `normalized <naming_>`_ version of the namespace e.g.
+  ``foo-bar``.
+* ``open``: This is a boolean indicating whether the namespace is
+  `open <open-namespaces_>`_.
+* ``parent``: This is the parent namespace if it exists. For example, if the
```

Comment:
With the change to grant hierarchy, this is the first place "parent namespace" and "child namespaces" are mentioned.  The PEP implies their meaning by way of example, but I think these terms should be explicitly defined, perhaps in the Terminology section.